### PR TITLE
Added missing Grub template info

### PR DESCRIPTION
**Pull request by lzap**
SSIA

### CODE REVIEW

File: plugins/foreman_discovery/nightly/index.md [hunk 1]
```diff
@@ -367,6 +366,14 @@ change the default entry from localboot entry to Discovery:
 
     ONTIMEOUT discovery
 
+Similar change is needed in *PXEGrub2 global default* template:
+
+    default=discovery
+
+The same way *PXEGrub1 global default* must be changed, but this is a legacy
+bootloader not in wide use anymore. In that case provide number as Grub1 does not
```
Comment (ekohl) [inline plugins/foreman_discovery/nightly/index.md]: the number?

File: plugins/foreman_discovery/nightly/index.md [hunk 2]
```diff
@@ -367,6 +366,14 @@ change the default entry from localboot entry to Discovery:
 
     ONTIMEOUT discovery
 
+Similar change is needed in *PXEGrub2 global default* template:
+
+    default=discovery
+
+The same way *PXEGrub1 global default* must be changed, but this is a legacy
```
Comment (ekohl) [inline plugins/foreman_discovery/nightly/index.md]: This doesn't read well to me. Perhaps `*PXEGrub1 global default* must be changed in the same way,` would be better?

**Comment by lzap** [comment]
Done thanks.

**Review by chris1984 — APPROVED** [review]
@lzap LGTM now that @ekohl comments have been addressed.

File: plugins/foreman_discovery/nightly/index.md [hunk 3]
```diff
@@ -367,6 +366,14 @@ change the default entry from localboot entry to Discovery:
 
     ONTIMEOUT discovery
 
+Similar change is needed in *PXEGrub2 global default* template:
+
+    default=discovery
+
+*PXEGrub1 global default* must be changed in the same way, but this is a legacy
+bootloader not in wide use anymore. In that case provide number as the Grub1
```
Comment (evgeni) [inline plugins/foreman_discovery/nightly/index.md]: what does `number` refer to? "the number of the menu entry"?

**Review by evgeni — CHANGES_REQUESTED** [review]

File: plugins/foreman_discovery/nightly/index.md (hunk 3 above)
Comment (lzap) [reply plugins/foreman_discovery/nightly/index.md]: Yes, grub1 only supports indexed selection.

**Comment by lzap** [comment]
Made an edit to make this more clear. But in 1.18+ this is not necessary - we have an administer-setting option for that now.

**Comment by lzap** [comment]
Hello?

File: plugins/foreman_discovery/nightly/index.md [hunk 4]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (ekohl) [inline plugins/foreman_discovery/nightly/index.md]: Isn't this needed to mark it as metadata?

Comment (lzap) [reply plugins/foreman_discovery/nightly/index.md]: Good catch that was an accident. Put back.

**Review by ekohl — APPROVED** [review]

File: plugins/foreman_discovery/nightly/index.md [hunk 5]
```diff
@@ -367,6 +367,15 @@ change the default entry from localboot entry to Discovery:
 
     ONTIMEOUT discovery
 
+Similar change is needed in *PXEGrub2 global default* template:
+
+    default=discovery
+
+*PXEGrub1 global default* must be changed in the same way, but this is a legacy
+bootloader not in wide use anymore. In that case provide number of the
```
Comment (ekohl) [inline plugins/foreman_discovery/nightly/index.md]:
```suggestion
bootloader not in wide use anymore. In that case provide the number of the
```

File: plugins/foreman_discovery/nightly/index.md [hunk 6]
```diff
@@ -367,6 +367,15 @@ change the default entry from localboot entry to Discovery:
...
+discovery menu entry (starting from 0) as the Grub1 does not accept menu
```
Comment (ekohl) [inline plugins/foreman_discovery/nightly/index.md]:
```suggestion
discovery menu entry (starting from 0) as Grub1 does not accept menu
```

**Comment by lzap** [comment]
Amended thanks.

**Comment by lzap** [comment]
Ping? Anyone?

**Review by evgeni — APPROVED** [review]

**Pull request merged**